folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -59,7 +59,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 
-public class MemcacheClientBuilder<V> {
+public class MemcacheClientBuilder<V extends Serializable> {
```

Comment:
This line can and should be reverted, to avoid breaking usages that do not use Serializable classes